optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc

```diff
@@ -114,7 +114,7 @@ Add the following code to your solver configuration:
     <solver xmlns="https://www.optaplanner.org/xsd/solver" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="https://www.optaplanner.org/xsd/solver https://www.optaplanner.org/xsd/solver/solver.xsd">
       <scoreDirectorFactory>
-        <constraintProviderClass>com.example.MyConstraintProvider</constraintProviderClass>
+        <constraintProviderClass>org.acme.schooltimetabling.solver.TimeTableConstraintProvider</constraintProviderClass>
```

Comment:
I kinda favor that docs example code typically copy paste something from a real example/quickstart, so it has a higher change of working and so users can look for it to see it in action.